latex/ledger-spec.tex

```diff
@@ -320,6 +414,50 @@ \section{UTxO}
   \label{fig:functions:deposits}
 \end{figure}
 
+\subsection{UTxO Transitions}
+\label{sec:state-trans-utxo-1}
+
+
+The types involved in defining a UTxO and its transitions are defined in
+Figure~\ref{fig:defs:utxo}. A transaction $\var{tx}$ (a term of type $\Tx$)
+contains a list of inputs (a list of terms of type $\TxIn$), and outputs
+(a finite map $\Ix \mapsto \TxOut$). Each input in the list
+corresponds to those $\TxOut$ terms in the outputs list indexed by the same
+$\Ix$ as in the second coordinate of the input. This way, a transaction can have
+inputs from multiple prior transactions and outputs to multiple addresses
+from each of the input transactions.
+
+The map $\var{txid}$ computes a hash of a given
+transaction to give its $\var{txid}$. A transaction can be uniquely identified
+by its $\var{txid}$ in a given context.
+
+Now, the actual type $\Tx$ of a transaction is likely to necessitate additional data
+in an actual implementation. For example, transaction fees (discussed below)
+or certificates to be applied
+may need to be part of the transaction data. Much of the time, however, when defining
+properties in this document, we want to refer to only the core of the transaction.
+For this reason, we make use here
+of the $\fun{txbody}$ map. This map returns only the data in the given
+transaction pertinent to the transfer of coins (i.e. the lists of inputs and outputs)
+
+The transaction fee $\fun{txfee}$ is the
+fee a transaction contributes to the system (this value depends only on
+the transaction itself).
+
+\begin{note}
+  Fees have no explicit output?
```

Comment:
correct. if the fee is explicitly specified in the transaction, then as long at the preservation of value condition is met (by the equality), then we do not need to specify an an explicit output for the fee